internal/app/gateway/command/run.go

```diff
@@ -25,6 +25,7 @@ func init() {
 	gatewayCommand.PersistentFlags().BoolVarP(&dev, "dev", "d", false, "run mashling in dev mode")
 	gatewayCommand.PersistentFlags().StringVarP(&configCache, "config-cache", "C", ".cache", "location of the configuration artifacts cache")
 	gatewayCommand.PersistentFlags().BoolVarP(&configCacheEnabled, "config-cache-enabled", "E", true, "cache post-processed configuration artifacts locally")
+	gatewayCommand.PersistentFlags().StringVarP(&pingPort, "ping-port", "p", "9090", "configure mashling gateway ping service port")
```

Comment:
Also, what happens if a user wants the ping service disabled? Maybe that requires a different flag (ping-enabled, defaulting to true maybe)? This will impact the gateway start and start logic too as the ping service is assumed to always be started, it appears.

Reply:
Agreed that there should be an option to disable ping.

Reply:
Agreed.